setup.py

```diff
@@ -112,7 +112,9 @@ def patch_yajl_sources():
     extra_sources.remove(os.path.join(yajl_sources, 'src', 'yajl_version.c'))
     extra_include_dirs = [yajl_sources, os.path.join(yajl_sources, 'src')]
     libs = []
-if embed_yajl or have_yajl:
+build_yajl_default = '1' if embed_yajl or have_yajl else '0'
+build_yajl = os.environ.get('IJSON_BUILD_YAJL', build_yajl_default) == '1'
```

Comment:
My only comment is that probably a better name would be `IJSON_BUILD_YAJL2C` or `IJSON_BUILD_YAJL2C_EXT` to indicate this refers to the ijson backend and not the yajl library itself.

I also just realised I'm not mentioning the `IJSON_EMBED_YAJL` environment variable anywhere in the documentation, but that's probably because there's no section on building the package. When/if I add one I'll surely mention this new variable as well.

Reply:
Oh, and would you mind rebasing too? Thanks!